lib/exercises.rb

```diff
@@ -2,19 +2,67 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n)
+# Space Complexity: O(n)
 
 def grouped_anagrams(strings)
```

Comment:
👍 Assuming the strings are small, this is correct, otherwise sorting them takes some time complexity... 

However look at how you're you're finding keys below, that's an O(n^2) solution and you might as well be using arrays.